supervision/detection/tools/smoother.py

```diff
@@ -0,0 +1,161 @@
+from collections import deque
+from typing import Optional
+
+import numpy as np
+
+from supervision.detection.core import Detections
+
+
+class Smoother:
```

Comment:
What do you think about changing the name to `DetectionsSmoother`?

Reply:
Updated in my latest commit.